.gitignore

```diff
@@ -16,3 +16,6 @@
 
 # Compiled files
 /dist
+
```

Comment:
can you remove this from here, please? I really want to avoid IDE/system specific stuff on project's gitignores